bedrock/firefox/templates/firefox/includes/download-button-thanks.html

```diff
@@ -0,0 +1,19 @@
+<div class="mzp-c-button-download-container c-button-download-thanks">
```

Comment:
You can already add a custom button class and ID to the link itself. I don't think it makes sense to have another set for the wrapper, so if it's more useful I'll move them both to the top HTML element?

Reply:
Ok, we need to have a custom button class on the link so we can do things like `mzp-t-secondary`. It might be potentially confusing to have a different param and class for the wrapper, so what I propose is to make the ID belong to the top level element. Does that work?

Reply:
ID on top-level +1.

I think that's best until we can have parent selectors in CSS.